src/databricks/labs/ucx/install.py

```diff
@@ -104,12 +105,51 @@ def run(self):
         self._run_configured()
 
     def _run_configured(self):
+        self._install_spark_config_for_hms_lineage()
         self._create_dashboards()
         self._create_jobs()
         readme = f'{self._notebook_link(f"{self._install_folder}/README.py")}'
         msg = f"Installation completed successfully! Please refer to the {readme} notebook for next steps."
         logger.info(msg)
 
+    def _install_spark_config_for_hms_lineage(self):
+        if (
+            self._prompts
+            and self._question(
```

Comment:
Check first if the script exists, before asking the question.

Reply:
Hi @FastLee - took care of this.